app/styles/recap-situation.css

```diff
@@ -0,0 +1,66 @@
+.recap-situation {
+    border-top: none;
```

Comment:
The difference is almost impossible to see. If you confirm, please just remove this instruction — this way we can drop the selector altogether  😃

(with `border-top: none`:
![capture d ecran 2017-06-13 a 22 31 02](https://user-images.githubusercontent.com/222463/27103167-1013a782-5088-11e7-8c1d-4db288d19745.png)
without:
![capture d ecran 2017-06-13 a 22 30 59](https://user-images.githubusercontent.com/222463/27103171-181dc840-5088-11e7-9bca-13c6077e58fb.png)
)